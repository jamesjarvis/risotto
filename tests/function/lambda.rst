printer := func (i int, s string) {
    println(i+s)
}

printer(42, " IT WORKS")

func printIt(printItPrinter func (i int, s string)) {
    printItPrinter(43, " IT REALLY WORKS")
}

printIt(printer)
