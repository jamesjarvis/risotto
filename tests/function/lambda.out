42 IT WORKS
43 IT REALLY WORKS
